bin/alluxio-masters.sh

```diff
@@ -30,6 +30,7 @@ ALLUXIO_LIBEXEC_DIR=${ALLUXIO_LIBEXEC_DIR:-${DEFAULT_LIBEXEC_DIR}}
 . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
 
 HOSTLIST=$(cat "${ALLUXIO_CONF_DIR}/masters" | sed  "s/#.*$//;/^$/d")
+ALLUXIO_LOGS_DIR=$(${BIN}/alluxio getConf alluxio.logs.dir)
```

Comment:
Consider this case. The user does not set the environment variable called ALLUXIO_LOGS_DIR. When ./bin/alluxio-masters.sh is invoked, in this script, the ALLUXIO_LOGS_DIR will initially equal an empty string. Thus we want to evaluate it first. We use the getConf utility because this value can be configured in alluxio-site.properties.